easybuild/easyconfigs/p/Pyomo/Pyomo-5.5.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'Bundle'
+
+name = 'Pyomo'
+version = '5.5.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.pyomo.org/'
+description = """ Pyomo is a Python-based open-source software package that supports a diverse set of optimization
+ capabilities for formulating and analyzing optimization models. """
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [('Python', '2.7.15')]
+
+exts_defaultclass = 'PythonPackage'
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
+
+exts_list = [
+    ('appdirs', '1.4.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/a/appdirs'],
+        'checksums': ['9e5896d1372858f8dd3344faf4e5014d21849c756c8d5701f78f8a103b372d92'],
+    }),
+    ('ply', '3.11', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/ply'],
+        'checksums': ['00c7c1aaa88358b9c765b6d3000c6eec0ba42abca5351b095321aef446081da3'],
+    }),
+    ('PyUtilib', '5.6.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyutilib'],
+        'checksums': ['6a21fccfe691c39566c0bb19b5c9aa11bca8b076aa6f1dbf21c11711f5105191'],
+        'use_pip': False,
```

Comment:
@smoors Please add a comment above this line referencing to https://github.com/PyUtilib/pyutilib/issues/39 to clarify why this is required.